client/my-sites/domains/domain-management/transfer/transfer-to-other-site/confirmation-dialog.jsx

```diff
@@ -28,8 +29,35 @@ class TransferConfirmationDialog extends React.PureComponent {
 		this.props.onConfirmTransfer( this.props.targetSite, closeDialog );
 	};
 
+	getMessage() {
+		const { domainName, isMappedDomain, targetSite, translate } = this.props;
+		const targetSiteTitle = get( targetSite, 'title', '' );
+		if ( isMappedDomain ) {
+			return translate(
+				'Do you want to transfer mapping of {{strong}}%(domainName)s{{/strong}} ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Do you want to transfer {{strong}}%(domainName)s{{/strong}} to site {{strong}}%(targetSiteTitle)s{{/strong}}?' )` *ES Score: 8*